model/event/subscriptions.go

```diff
@@ -38,6 +39,7 @@ type Subscription struct {
 	Selectors      []Selector    `bson:"selectors,omitempty"`
```

Comment:
I'm probably going to rename the type field above to ResourceType to be more accurate. Can you make the json tags for the apimodels reflect this?

Reply:
I was referring to Subscription (and thus APISubscription)

Reply:
Could you change the API tag when you make your change? I'd like to keep them consistent